agent/taskresource/envFiles/envfile.go

```diff
@@ -0,0 +1,198 @@
+package envFiles
```

Comment:
missing copy right headers. same for all other files

Reply:
added